### ISSUE TITLE
New website needs to support some kind of documentation search

### ISSUE THREAD

**Issue by kltm** [geneontology/go-site#977]
Currently, the new website does not have any method of searching within the site/documentation. This is a very common use case and will need to be supported for release.

Most easily, this could be done with a custom search box from google, etc. It may be nice to supply a site.xml as well.

Other (more complicated) methods could be https://lunrjs.com/ , compiled search in jekyll, etc. It may be better to work on something more complete later on in a different ticket.

**Comment by lpalbou** [comment]
@kltm this is wrong, the [search page](https://geneontology.github.io/search) is working since October and you can access it from the Help / Search Documentation menu.

Everything is indexed/reindexed with a single command:
> ALGOLIA_API_KEY=xxx bundle exec jekyll algolia

And that's it.

**Comment by lpalbou** [comment]
Additionally, I don't understand why this post is here and not in the geneontology.github.io repo ?

**Comment by kltm** [comment]
I don't see a search box on the landing page for https://test.geneontology.io/
